earth_enterprise/src/fusion/gepublish/gepublishmanagerhelper.h

```diff
@@ -20,9 +20,9 @@
 #ifndef GEO_EARTH_ENTERPRISE_SRC_FUSION_GEPUBLISH_GEPUBLISHMANAGERHELPER_H_
 #define GEO_EARTH_ENTERPRISE_SRC_FUSION_GEPUBLISH_GEPUBLISHMANAGERHELPER_H_
 
-#include <Python.h>
-#include <cstddef>
+#include <Python.h>  // Note: should be first in include list.
```

Comment:
because of warning or errors in compilation ?

Reply:
I don't know. I just preserved the comment from the old source.

I think `Python.h` is a meta-include file that includes other files based on configuration options, so you probably don't want to already include something that it conditionally includes (or, replaces with some other include).

Reply:
ok